internal/examples/json-keyvalue/server/main.go

```diff
@@ -38,7 +39,10 @@ import (
 	"go.uber.org/yarpc/yarpcerrors"
 )
 
-var flagInbound = flag.String("inbound", "", "name of the inbound to use (http/tchannel/grpc)")
+var (
+	flagSet     = flag.NewFlagSet("server", flag.ExitOnError)
```

Comment:
since these are internal we could use `flag.PanicOnError` to get a `... panic with a descriptive error` instead of just an `os.Exit(2)`
ref: https://golang.org/pkg/flag/#ErrorHandling